pkg/apis/pravega/v1alpha1/pravegacluster_types.go

```diff
@@ -189,6 +189,7 @@ type TLSPolicy struct {
 type StaticTLS struct {
 	ControllerSecret   string `json:"controllerSecret,omitempty"`
 	SegmentStoreSecret string `json:"segmentStoreSecret,omitempty"`
+	CaBundle           string `json:"caBundle,omitempty"`
```

Comment:
This CaBundle is specifically for ECS. Could we give it a name that indicates this?
Alternatively it could be placed under tier2/ecs in the spec.

Reply:
David specifically required this name:
"yes, but just call ecsSecret caBundle that way it can be used in future for any other external systems the segmentstore may need to contact"